proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/HubResponseAppRuleTests.java

```diff
@@ -48,6 +48,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
+import static org.opensaml.saml.saml2.core.StatusCode.*;
```

Comment:
Since it's in a test it's probably not such a big deal but we should avoid `*` imports.